src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  box-sizing: border-box;
+  color: $main-text-color;
```

Comment:
Ensure that the SCSS variable `$main-text-color` is defined in your stylesheets. If not, you need to define it or replace it with a valid color value.